class NotFoundError(Exception):
    pass


class S3Error(Exception):
    pass